content/docs/tasks/traffic-management/request-routing/index.md

```diff
@@ -20,14 +20,39 @@ This task shows you how to configure dynamic request routing based on weights an
 
 ## Content-based routing
 
+The Bookinfo sample deploys multiple versions of each microservice, so we will start by creating DestinationRules
```

Comment:
Please avoid using "we". That is, it should be something like "so you will start by"

Reply:
done